service/gradle/dependencies.gradle

```diff
@@ -116,6 +116,8 @@ dependencies {
 
   // pact
   testImplementation 'au.com.dius.pact.consumer:junit5:4.1.7'
+  testImplementation 'au.com.dius.pact.provider:junit5:4.1.7'
+  testImplementation 'au.com.dius.pact.provider:junit5spring:4.1.7'
```

Comment:
It's possible we could have a more recent version.  I know there were weirdnesses with this, did you experiment with different versions here?

Reply:
Ok I bumped the dependency up to the one mentioned in the thread. That swapped out the implementation for a kotlin-based one which moved a bunch of stuff around.   Had to fiddle with the environment properties, and stop directly using a UUID constant which is no longer visible due to being buried in a kotlin companion object.

Reply:
Also needed to upgrade existing pact _consumer_ tests to comply with a new API signature convention for V4 Pacts: https://github.com/DataBiosphere/terra-workspace-manager/pull/1511/commits/2975f3797121e9a808b93f22d3e9bd5fbff9af17

Reply:
This turned into a major yakshave, but I think I am coming out the other side with the following prerequesite PR: https://github.com/DataBiosphere/terra-workspace-manager/pull/1517